supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -359,6 +359,50 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[[inference]]
+==== image:images/yes.png[yes] inference (noun)
+*Description*: Inference is when a model can process, deduct, and produce outputs based on input data. The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model.
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[`InferenceService`]]
+==== image:images/yes.png[yes] `InferenceService` (noun)
```

Comment:
We don't use markup in the headings. Just in the description below. Though to be honest, I don't think we've historically really put object names in as glossary entries. Is this always written this way, or would it sometimes be written generically as the "inference service"?

If yes, then I would put "inference service" as the glossary entry, and then say something in the description like, if you're talking about the actual object, then write it as `InferenceService`.

For a glossary entry in a similar situation, see "pod", "persistent volume claim", "service", etc.

```suggestion
==== image:images/yes.png[yes] InferenceService (noun)
```